modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/util/JournalUtil.java

```diff
@@ -234,6 +238,21 @@ public static long getPreviewPlid(
 				LayoutConstants.DEFAULT_PARENT_LAYOUT_ID);
 		}
 
+		PermissionChecker permissionChecker =
+			PermissionCheckerFactoryUtil.create(themeDisplay.getUser());
+
+		if (!LayoutPermissionUtil.contains(
+				permissionChecker, layout, ActionKeys.VIEW)) {
+
+			layout = _partialSearchLayout(
+				themeDisplay, permissionChecker, false);
+
+			if (layout == null) {
+				layout = _partialSearchLayout(
+					themeDisplay, permissionChecker, true);
+			}
+		}
+
```

Comment:
I think you checked the permissions of the first layout twice.

Reply:
I got it. I'll ignore it by Id checking.